test/shared/src/test/scala/monocle/std/StringsSpec.scala

```diff
@@ -1,11 +1,29 @@
 package monocle.std
 
+import java.util.UUID
+
 import monocle.MonocleSuite
 import monocle.function.Plated._
 import monocle.law.discipline.function._
 import monocle.law.discipline.{IsoTests, PrismTests, TraversalTests}
+import org.scalacheck.Arbitrary
+
+import scalaz.Equal
 
 class StringsSpec extends MonocleSuite {
+  implicit def arbitraryUUID: Arbitrary[UUID] = Arbitrary(UUID.randomUUID)
+
+  implicit def equalUUID: Equal[UUID] = new Equal[UUID] {
+    override def equal(a1: UUID, a2: UUID): Boolean = a1 == a2
+  }
+
+  implicit def arbitraryUUIDtoUUID: Arbitrary[UUID => UUID] = Arbitrary(
```

Comment:
it should be enough to have a `Cogen[UUID]`. Btw, you can move this definition to `TestInstances`